FirebasePerformance/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Pending
 * Create a random number of delay for remote config fetch during app starts.
 * Fix log spamming when Firebase Performance is disabled. (#8423)
+* Fix heap-heap-buffer overflow when decoding strings. (#8628)
```

Comment:
I thought there were 2 bugs related to this. Can you tag both of them here?

Reply:
Done.